test/passes/set-globals.wast

```diff
@@ -0,0 +1,7 @@
+(module
+ ;; an imported i32
+ (import "env" "in" (global $foo i32))
+
+ ;; a defined i64
+ (global $bar i64 (i64.const 1234))
```

Comment:
Since it also support creating a global, we can have a test case for it too I think?

Reply:
Oh, oops, that was a bug. I intended it to skip missing globals. Fixing now.

(Adding globals is possible, but we'd need to be given the type too, and not sure there is a use case.)